### PR TITLE
MES-2901 - View Test Result Prep - Part 1

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Created a Cat B Version of the page

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/page-names.constants.ts [hunk 1]
```diff
@@ -42,4 +42,5 @@ export const CAT_BE = {
   REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
   NON_PASS_FINALISATION_PAGE: 'NonPassFinalisationCatBEPage',
   VEHICLE_CHECKS_MODAL: 'VehicleChecksCatBEModal',
+  VIEW_TEST_RESULT_PAGE:'ViewTestResultCatBPage',
```
Comment (rossfauldsbjss) [inline src/pages/page-names.constants.ts:45]: Cat B page not BE

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/page-names.constants.ts (hunk 1 above)
Comment (TomBillingtonUK) [reply src/pages/page-names.constants.ts:45]: Cat BE page doesn't exist yet so will update this again when I add it .

File: src/pages/test-report/cat-be/__tests__/test-report.cat-be.page.spec.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ import { candidateMock } from '../../../../modules/tests/__mocks__/tests.mock';
 import { UncoupleRecoupleComponent } from '../components/uncouple-recouple/uncouple-recouple';
 import { ReverseLeftComponent } from '../components/reverse-left/reverse-left';
 import { ReverseLeftPopoverComponent } from '../components/reverse-left-popover/reverse-left-popover';
-import { VehicleChecksComponent } from '../components/vehicle-checks/vehicle-checks'
+import { VehicleChecksComponent } from '../components/vehicle-checks/vehicle-checks';
```
Comment (Cullima) [inline src/pages/test-report/cat-be/__tests__/test-report.cat-be.page.spec.ts:43]: thanks!

**Review by Cullima — APPROVED** [review]

**Pull request merged**